application/forms/Command/Object/DeleteDowntimeForm.php

```diff
@@ -25,11 +27,29 @@ protected function assembleElements()
 
     protected function assembleSubmitButton()
     {
+        $isDisabled = false;
+        foreach ($this->getObjects() as $downtime) {
+            if ($downtime->scheduled_by) {
+                $downtimeName = $downtime->name;
+                $scheduledBy = $downtime->scheduled_by;
+                $isDisabled = true;
```

Comment:
I don't think it's a good idea to disable the button just because there's one downtime that cannot be removed. Just disable it if *all* downtimes cannot be removed.

Then in `getCommand` only return a command if removal is possible.